pyscf/pbc/dft/multigrid/multigrid_pair.py

```diff
@@ -1038,10 +1038,11 @@ def get_nuc_nuc_grad(mydf, dm, kpts=None):
     charge = -cell.atom_charges()
     grad = np.zeros((cell.natm,3))
     # TODO improve performance
+    rhoG = _eval_rhoG(mydf, dms, hermi=1, kpts=kpts).ravel()
```

Comment:
Should this be computed as `_eval_rhoG(mydf, dms.conj(), hermi=1, kpts=kpts).ravel().conj()`? The `_get_j_pass2` applies a IFFT on vG while the _eval_rhoG applies FFT on rhoR.  Here, to compute IFFT(rhoR) = conj(IFFT(rhoR.conj()))

Reply:
It should be the same. If `rhoG = _eval_rhoG().conj()`, then the following line will be `vG = -1j * exp(-1j * dot(Gv, R)) * charge * coulG * Gv`